File: 프로그래머스/1/12931. 자릿수 더하기/자릿수 더하기.java
import java.util.*;

public class Solution {
    public int solution(int n) {
        int answer = 0;
        String text = String.valueOf(n);
    
        for (int i = 0; i < text.length(); i++) {
            answer += Character.getNumericValue(text.charAt(i));
        }
        return answer;
    }
}

File: 프로그래머스/1/12931. 자릿수 더하기/자릿수 더하기.cpp
#include <iostream>
#include <algorithm>

using namespace std;
int solution(int n)
{
    int answer = 0;
    string str = to_string(n);
    for (int i = 0; i < str.size(); i++) {
        answer += n % 10;
        n = (int) n / 10;
    }
    return answer;
}